internal/usecase/create_todo.go

```diff
@@ -16,6 +18,7 @@ type CreateTodoUsecase struct {
 
 func (ctiu *CreateTodoUsecase) CreateTodo(input CreateTodoInput) error {
 	todo := entity.Todo{
+		ID:       uuid.New().String(),
```

Comment:
In [doc](https://pkg.go.dev/github.com/google/uuid#New)

```
func New() UUID

New creates a new random UUID or panics. New is equivalent to the expression
```

It says it can panic, have you caught it somewhere? Or you just leave it be? If it panics then the program will crash?

Reply:
well, it won't crash the whole program, but the request is failed unexpectedly